omod/src/main/java/org/openmrs/module/pihcore/page/controller/visit/VisitPageController.java

```diff
@@ -66,7 +66,7 @@ else if (encounter != null) {
         model.addAttribute("encounterType", encounterType);
         model.addAttribute("suppressActions", (suppressActions != null) ? suppressActions : false);
         model.addAttribute("encounter", encounter);
-        model.addAttribute("locale", uiSessionContext.getLocale());
+        model.addAttribute("locale", uiSessionContext.getLocale().getLanguage());
```

Comment:
Can we add a in-line comment with the reasoning behind this (MessagesController currently only supports language?)

Reply:
Right.  I understand the issue you are facing, I just don't understand why the solution is to remove country here, rather than to add country support into the Message Controller.  Isn't our goal to support proper localization?

Reply:
Just got off a call with @mogoodrich who recalls that the last time he tried that, it introduced other bugs. It is, of course, the right thing to do. Can I ticket it and merge this in, @mseaton ? Would be great to get Peru as un-blocked as possible.

Reply:
works for me...

Reply:
Sounds good.